common/prettify.pm

```diff
@@ -1746,7 +1746,7 @@ sub Test_Handler ($)
         || $s =~ m/Don't know how to make check/
         || $s =~ m/Could not open/
         || $s =~ m/Errno::ENOENT/
-        || $s =~ m/No such file or directory/
+        || ($s =~ m/No such file or directory/ && not ($s =~ m/LLVMSymbolizer: error reading file: No such file or directory/))
```

Comment:
```suggestion
        || ($s =~ m/No such file or directory/ && $s !~ m/LLVMSymbolizer: error reading file:/)
```
If it gets to the second operand of the `&&` it already matched `No such file...`